core/lib/dal/migrations/20240523085604_add_tee_proof_generation_details_table.down.sql

```diff
@@ -1,4 +1,6 @@
+DROP INDEX IF EXISTS idx_tee_proofs_status_prover_taken_at;
```

Comment:
You should not edit the migrations, because some of them are already applied on the existing environments.
Instead, you need to create a new migration (see `cargo sqlx` CLI docs for that).

Reply:
Oh, I naively assumed `sqlx` could automatically handle all the migration steps based on the modifications I made to the SQL table definitions. I'm not sure if I'm doing something wrong or if it's just a real pain to create the right SQL queries for making or undoing changes in the existing SQL tables in the `.up.sql` and `.down.sql` files, but here they are: https://github.com/matter-labs/zksync-era/pull/2474/commits/6e5d39f67318784b8998996dbd7d7e6a9d5fe773. @popzxc please let me know if that looks good enough.